_pytest/logging.py

```diff
@@ -356,12 +357,30 @@ class _LiveLoggingStreamHandler(logging.StreamHandler):
     """
     Custom StreamHandler used by the live logging feature: it will write a newline before the first log message
     in each test.
+
+    During live logging we must also explicitly disable stdout/stderr capturing otherwise it will get captured
+    and won't appear in the terminal.
     """
 
+    def __init__(self, terminal_reporter, capture_manager):
+        """
+        :param _pytest.terminal.TerminalReporter terminal_reporter:
+        :param _pytest.capture.CaptureManager capture_manager:
+        """
+        logging.StreamHandler.__init__(self, stream=terminal_reporter)
+        self.capture_manager = capture_manager
+        self._first_record_emitted = False
```

Comment:
This needs to be set to False at the beginning of every test. Otherwise I get the following output when the multiple tests are run:

```
(pytest_venv) thomas-XPS-13-9360 :: ~gr/pytest_logging_tests » pytest test3.py -o log_cli=true -o log_level=DEBUG 
================================================================= test session starts ==================================================================
platform linux2 -- Python 2.7.12, pytest-3.3.3.dev56+g3e93eac, py-1.5.2, pluggy-0.6.0 -- /home/thomas/gitrepos/pytest_venv/bin/python
cachedir: .cache
rootdir: /home/thomas/gitrepos/pytest_logging_tests, inifile: pytest.ini
plugins: xdist-1.20.1, sphinx-0.2, forked-0.2, cov-2.5.1, hypothesis-3.40.1
collected 3 items                                                                                                                                      

test3.py::test_captured_log_report1 
----------------------------------------------------------------------- live log -----------------------------------------------------------------------
test3.py                     7 INFO     INFO message won't be shown
test3.py                     8 WARNING  WARNING message will be shown
FAILED                                                                                                                                           [ 33%]
test3.py::test_captured_log_report2 test3.py                    13 INFO     INFO message won't be shown
test3.py                    14 WARNING  WARNING message will be shown
PASSED                                                                                                                                           [ 66%]
test3.py::test_captured_log_report3 test3.py                    18 INFO     INFO message won't be shown
test3.py                    19 WARNING  WARNING message will be shown
PASSED                                                                                                                                           [100%]
.......
```

Reply:
Oops indeed, thanks!